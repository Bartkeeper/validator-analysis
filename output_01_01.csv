üê†stake.fish,0.057308,115000000000uatom,11506112390721uatom
Coinbase Custody,0.055809,2000000uatom,11205180798472uatom
DokiaCapital,0.052702,350000000000uatom,10581304494805uatom
Zero Knowledge Validator (ZKV),0.046899,9000000uatom,9416116634579uatom
SG-1,0.046232,2642000000uatom,9282300024551uatom
Binance Node,0.040791,6999300081uatom,8189764203708uatom
Polychain,0.034379,1000000uatom,6902429008972uatom
Everstake,0.032885,277900000uatom,6602603135046uatom
GAME,0.032562,1000000uatom,6537645169535uatom
Paradigm,0.032255,2000000uatom,6475951643754uatom
Sikka,0.029921,19896210513uatom,6007322623909uatom
Figment,0.026287,0uatom,5277816195102uatom
Cosmostation,0.018806,23117172506uatom,3775699235872uatom
Binance Staking,0.018574,0uatom,3729282794461uatom
P2P.ORG - P2P Validator,0.018451,4963825622uatom,3704563350022uatom
Chorus One,0.017466,1000000uatom,3506684420871uatom
Kraken,0.017408,9000000uatom,3495126802818uatom
MultiChain ventures,0.016278,19998000uatom,3268314872102uatom
Informal Systems,0.015608,8527960633uatom,3133699899121uatom
NO! Fee to 2025 üí∏ | melea.xyz,0.014987,30111111uatom,3009076910058uatom
Provalidator,0.013886,10000000uatom,2787961319424uatom
Citadel.one SuperApp,0.013646,2194000000uatom,2739854098840uatom
StakeLab,0.012744,1055000uatom,2558700819484uatom
Certus One,0.011913,63620000uatom,2391750519095uatom
zugerselfdelegation,0.011046,1000000uatom,2217681861526uatom
hashtower,0.010568,564536913686uatom,2121840452500uatom
Allnodes.com ‚ö°Ô∏è 0% fee,0.010460,1000000uatom,2100056917865uatom
Imperator.co,0.010399,3000000uatom,2087837445882uatom
BouBouNode,0.009738,17782112762uatom,1955090906610uatom
iqlusion,0.009334,20000000000uatom,1873981047257uatom
Swiss Staking,0.007831,1000000uatom,1572321127454uatom
CoinoneNode,0.007697,138381363196uatom,1545432466715uatom
Blockpower,0.007588,4039500002uatom,1523418884385uatom
Forbole,0.007440,0uatom,1493824287793uatom
Staking Facilities,0.006920,290799082078uatom,1389357337489uatom
strangelove-ventures,0.006709,3900008uatom,1347095144578uatom
DACM,0.006602,90990900uatom,1325473371153uatom
Staked,0.006377,103226uatom,1280387990575uatom
Node Guardians,0.005817,541037540uatom,1167970858537uatom
B-Harvest,0.005817,999909uatom,1167945868240uatom
ChainLayer,0.005630,60000000008uatom,1130302350524uatom
dForce,0.005422,1000000uatom,1088697362608uatom
Post Road,0.005011,1uatom,1006071626511uatom
ShapeShift DAO,0.005004,25000000uatom,1004703342857uatom
Stakecito,0.004958,1000000000uatom,995449689858uatom
DelegaNetworks ‚àû,0.004186,11645000000uatom,840393626999uatom
Ledger,0.004039,1000000uatom,810840649610uatom
IRISnet-Bianjie,0.003850,59700000004uatom,772925073584uatom
KalpaTech,0.003711,700000000uatom,745153979560uatom
OKEx Pool,0.003503,5000uatom,703345429840uatom
POSTHUMAN Íùè DVS,0.003322,2000000uatom,666881841389uatom
Atomic Nodes,0.003313,999900uatom,665256809960uatom
üá®üá≠ Vortex.live üü¢,0.003251,20000000uatom,652799393280uatom
0base.vc,0.003207,55619357891uatom,643908665593uatom
Blockdaemon,0.003164,10000000uatom,635324412917uatom
üåê KysenPool Sky ‚òÅÔ∏è ,0.002934,500000000uatom,588989018106uatom
ATEAM,0.002909,2299080uatom,584029612525uatom
Latent Iron,0.002871,2000000uatom,576359979213uatom
Nocturnal Labs,0.002846,1999900uatom,571413108403uatom
Umbrella ‚òî,0.002821,665933400uatom,566433366911uatom
AUDIT.one,0.002798,2600008547uatom,561695303518uatom
InfStones,0.002768,1722777uatom,555689328826uatom
Simply Staking,0.002724,22343863799uatom,547000788839uatom
S16 Research Ventures,0.002680,1499400090uatom,537999651149uatom
WeStaking,0.002425,36000000000uatom,486843460521uatom
KuCoin,0.002345,1000001uatom,470805605924uatom
HyperblocksPro,0.002313,461811339241uatom,464339489486uatom
01node,0.002301,22964436055uatom,462083140191uatom
Upbit Staking,0.002214,23551000000uatom,444554986687uatom
Ubik Capital 0%Fee,0.002213,2335000000uatom,444375610315uatom
e-Money.com // validator.network,0.002175,9272040049uatom,436652195882uatom
Â§©ÁÖß‚òÄ,0.002141,426000000uatom,429873917676uatom
stakezone,0.002125,56034270398uatom,426655205338uatom
"Frens (ü§ù,ü§ù)",0.002054,1181271448uatom,412443886409uatom
"	EZStaking.io",0.002023,740632434uatom,406164639192uatom
Cypher Core,0.002010,0uatom,403538393089uatom
" 0% Fee üåª Sunflower",0.001935,1000000uatom,388514262069uatom
Onbloc Node,0.001823,1000000uatom,366044401860uatom
#decentralizehk - DHK dao,0.001743,60621600uatom,350034088283uatom
BlockHunters üéØ,0.001707,80000000uatom,342807631993uatom
SmartNodes,0.001703,83021467uatom,341920970488uatom
FreshATOMS.com,0.001638,6719695267uatom,328852651023uatom
Stakewolle.com | Auto-compound,0.001610,1000000uatom,323180519184uatom
HashQuark,0.001550,7000000uatom,311286633648uatom
nylira.net,0.001549,26568361885uatom,311020465472uatom
" polkachu.com",0.001524,1477001060uatom,305918248830uatom
Notional,0.001496,1000000uatom,300378715615uatom
Golden Ratio Staking,0.001480,120300000uatom,297198592152uatom
cosmosgbt,0.001465,1999800uatom,294144456701uatom
RockawayX Infra,0.001463,11000000uatom,293704767416uatom
DeFi Wallet,0.001451,2000000uatom,291305000509uatom
" Lavender.Five Nodes üêù",0.001388,120000000uatom,278673095558uatom
Witval,0.001348,12391551684uatom,270585466384uatom
pe4x72,0.001241,100000000015uatom,249224310013uatom
" WhisperNodeü§ê",0.001146,62110000uatom,230130003333uatom
Stakin,0.001089,5792397000uatom,218635243285uatom
StakeWithUs,0.001064,2487000000uatom,213553451083uatom
COS_Validator,0.001008,2000000uatom,202425123684uatom
üê°grant.fish,0.000995,100000000000uatom,199775262375uatom
Binary Holdings,0.000983,445000000uatom,197383160476uatom
Huobi,0.000970,1000000uatom,194760716362uatom
ChainUp,0.000938,1000000uatom,188230204780uatom
Twinstake-Validator,0.000925,1000000uatom,185755532926uatom
in3s.com,0.000912,10716657840uatom,183100587836uatom
Just-Mining,0.000907,1998800uatom,182167754449uatom
Smart Stake,0.000903,1556000000uatom,181397094219uatom
Stakely.io,0.000900,63000000uatom,180670683725uatom
Citizen Cosmos,0.000894,49267145uatom,179532235344uatom
PUPM√òS,0.000861,10000000uatom,172892376706uatom
Oni ‚õ©Ô∏è,0.000821,107136965uatom,164902841521uatom
coinhall.org üìàüîÑ,0.000813,1000000uatom,163239661482uatom
Ztake.org,0.000808,149985000uatom,162225669819uatom
Klub Staking,0.000808,1000000uatom,162187498756uatom
cros-nest,0.000785,1147900001uatom,157593863825uatom
jabbey,0.000777,0uatom,156083335760uatom
Easy 2 Stake,0.000775,9678026666uatom,155649559276uatom
GATA DAO,0.000765,77000000uatom,153545115567uatom
commercio.network,0.000760,3948000000uatom,152628822290uatom
NEOPLY,0.000748,1000000uatom,150112149979uatom
Blocks United | blocksunited.com,0.000746,499950008uatom,149809845817uatom
KIRA Staking,0.000728,1000000uatom,146253076534uatom
Silk Nodes,0.000701,1000000uatom,140764305376uatom
CryptoCrew Validators ‚úÖ,0.000685,1030750000uatom,137584850857uatom
AIR DROP STATION,0.000685,1000000uatom,137475046964uatom
Moonstake,0.000684,999600uatom,137391175349uatom
Stargaze,0.000681,25133644555uatom,136741586270uatom
CEX.IO ‚ù§Ô∏èüá∫üá¶,0.000668,1000000uatom,134117943652uatom
BitValidator,0.000665,1000000uatom,133592971717uatom
danku_zone w/ DAIC,0.000665,1100000uatom,133479468252uatom
blockscape,0.000650,6872095134uatom,130463544365uatom
Ping,0.000645,15600000000uatom,129483159829uatom
Castlenode,0.000624,1000000uatom,125311684334uatom
debo-validator,0.000612,24329999929uatom,122817085294uatom
Chainflow,0.000602,35617872936uatom,120917131899uatom
Any Labs,0.000601,999700uatom,120660266261uatom
Terra Nodes,0.000598,1880733950uatom,120139561104uatom
MANTRA DAO,0.000563,887822410uatom,113132895462uatom
Chill Validation,0.000550,380654867uatom,110509281486uatom
Fanfury | fury.fan | Stake to Win,0.000543,1000000uatom,109046490818uatom
Coinbase Cloud,0.000539,18960092871uatom,108301887182uatom
IcyCRO üßä,0.000523,2883465567uatom,105004021733uatom
stake.systems,0.000518,5112000000uatom,103997758848uatom
zoomerlabs,0.000506,502000000uatom,101594685151uatom
Compass,0.000481,15267561312uatom,96567502823uatom
a41,0.000470,70983000000uatom,94424346708uatom
TienThuatToan Capital,0.000470,1000000uatom,94394747045uatom
StakeSeeker by BTCS,0.000465,100000uatom,93339107961uatom
CROSSTECH,