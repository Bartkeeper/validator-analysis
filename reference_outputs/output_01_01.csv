üê†stake.fish,0.057216,115000000000uatom,11490491374222uatom
Coinbase Custody,0.055802,2000000uatom,11206569965921uatom
DokiaCapital,0.052690,350000000000uatom,10581519274435uatom
Zero Knowledge Validator (ZKV),0.046888,9000000uatom,9416249872613uatom
SG-1,0.046268,2642000000uatom,9291805570485uatom
Binance Node,0.040903,6999300081uatom,8214368418885uatom
Polychain,0.034389,1000000uatom,6906157507094uatom
Everstake,0.032711,277900000uatom,6569222905266uatom
GAME,0.032591,1000000uatom,6545119169576uatom
Paradigm,0.032247,2000000uatom,6475951643754uatom
Sikka,0.029904,19896210513uatom,6005433062319uatom
Figment,0.026379,0uatom,5297636494494uatom
Cosmostation,0.018815,23117172506uatom,3778540170703uatom
Binance Staking,0.018606,0uatom,3736515525872uatom
P2P.ORG - P2P Validator,0.018450,4963825622uatom,3705147534388uatom
Chorus One,0.017460,1000000uatom,3506386072565uatom
Kraken,0.017404,9000000uatom,3495124571762uatom
MultiChain ventures,0.016279,19998000uatom,3269323228906uatom
Informal Systems,0.015562,8571475098uatom,3125246938667uatom
NO! Fee to 2025 üí∏ | melea.xyz,0.015054,30111111uatom,3023162363108uatom
Provalidator,0.013924,10000000uatom,2796319516043uatom
Citadel.one SuperApp,0.013635,2194000000uatom,2738332762063uatom
StakeLab,0.012748,1055000uatom,2560107396723uatom
Certus One,0.011909,63620000uatom,2391605486725uatom
zugerselfdelegation,0.011022,1000000uatom,2213567866526uatom
hashtower,0.010567,564536913686uatom,2122123612119uatom
Allnodes.com ‚ö°Ô∏è 0% fee,0.010492,1000000uatom,2107165687138uatom
Imperator.co,0.010431,3000000uatom,2094774684982uatom
BouBouNode,0.009736,17782112762uatom,1955184710910uatom
iqlusion,0.009333,20000000000uatom,1874385861066uatom
Swiss Staking,0.007918,1000000uatom,1590072190069uatom
CoinoneNode,0.007701,138381363196uatom,1546539903666uatom
Blockpower,0.007589,4039500002uatom,1524080259532uatom
Forbole,0.007437,0uatom,1493620925403uatom
Staking Facilities,0.006918,290799082078uatom,1389376818693uatom
strangelove-ventures,0.006782,3900008uatom,1361914252990uatom
DACM,0.006600,90990900uatom,1325519620271uatom
Staked,0.006376,103226uatom,1280390728490uatom
Node Guardians,0.005831,541037540uatom,1170952136473uatom
B-Harvest,0.005818,999909uatom,1168412151752uatom
ChainLayer,0.005628,60000000008uatom,1130331847615uatom
dForce,0.005427,1000000uatom,1089836754766uatom
Post Road,0.005010,1uatom,1006163426511uatom
ShapeShift DAO,0.005003,25000000uatom,1004727433860uatom
Stakecito,0.004973,1000000000uatom,998625405910uatom
DelegaNetworks ‚àû,0.004188,11645000000uatom,840980859281uatom
Ledger,0.004105,1000000uatom,824371053328uatom
IRISnet-Bianjie,0.003849,59700000004uatom,772921229384uatom
KalpaTech,0.003711,700000000uatom,745183530276uatom
OKEx Pool,0.003502,5000uatom,703355429840uatom
POSTHUMAN Íùè DVS,0.003324,2000000uatom,667535742900uatom
Atomic Nodes,0.003322,999900uatom,667128992724uatom
üá®üá≠ Vortex.live üü¢,0.003251,20000000uatom,652828495070uatom
0base.vc,0.003193,55619357891uatom,641252631784uatom
üåê KysenPool Sky ‚òÅÔ∏è ,0.002936,500000000uatom,589618705065uatom
ATEAM,0.002908,2299080uatom,584072753951uatom
Latent Iron,0.002870,2000000uatom,576359979213uatom
Nocturnal Labs,0.002845,1999900uatom,571413108403uatom
Umbrella ‚òî,0.002820,665933400uatom,566418488918uatom
AUDIT.one,0.002786,2600008547uatom,559503471414uatom
Blockdaemon,0.002773,10000000uatom,556957166901uatom
InfStones,0.002768,1722777uatom,555844094101uatom
Simply Staking,0.002728,22343863799uatom,547786424461uatom
S16 Research Ventures,0.002679,1499400090uatom,538009938425uatom
WeStaking,0.002433,36000000000uatom,488569530842uatom
KuCoin,0.002344,1000001uatom,470801152377uatom
HyperblocksPro,0.002312,461811339242uatom,464309720104uatom
01node,0.002302,22964436055uatom,462307422566uatom
Upbit Staking,0.002234,23551000000uatom,448554986687uatom
Ubik Capital 0%Fee,0.002209,2335000000uatom,443571441304uatom
e-Money.com // validator.network,0.002174,9272040049uatom,436653213415uatom
Â§©ÁÖß‚òÄ,0.002139,426000000uatom,429584157956uatom
stakezone,0.002125,56034270398uatom,426675704970uatom
"Frens (ü§ù,ü§ù)",0.002059,1182351408uatom,413475049314uatom
"	EZStaking.io",0.002024,740632434uatom,406438800656uatom
Cypher Core,0.002010,0uatom,403575636939uatom
" 0% Fee üåª Sunflower",0.001955,1000000uatom,392569273281uatom
Onbloc Node,0.001820,1000000uatom,365543285204uatom
#decentralizehk - DHK dao,0.001746,60621600uatom,350641040989uatom
SmartNodes,0.001709,83021467uatom,343285057943uatom
BlockHunters üéØ,0.001708,80000000uatom,342963067857uatom
FreshATOMS.com,0.001638,6719695267uatom,329010534451uatom
Stakewolle.com | Auto-compound,0.001623,1000000uatom,325940623458uatom
HashQuark,0.001550,7000000uatom,311306298562uatom
nylira.net,0.001549,26568361885uatom,311020465472uatom
" polkachu.com",0.001525,1482023167uatom,306184748673uatom
Notional,0.001498,1000000uatom,300747359964uatom
Golden Ratio Staking,0.001483,120300000uatom,297862076278uatom
cosmosgbt,0.001465,1999800uatom,294145120341uatom
RockawayX Infra,0.001463,11000000uatom,293709934918uatom
DeFi Wallet,0.001454,2000000uatom,292009847645uatom
" Lavender.Five Nodes üêù",0.001390,120000000uatom,279190634055uatom
Witval,0.001353,12391551684uatom,271718159841uatom
pe4x72,0.001241,100000000015uatom,249224310040uatom
"  WhisperNodeü§ê",0.001147,62110000uatom,230412338105uatom
Stakin,0.001088,5792397000uatom,218401641139uatom
StakeWithUs,0.001066,2487000000uatom,214025114655uatom
COS_Validator,0.001008,2000000uatom,202425133557uatom
üê°grant.fish,0.000995,100000000000uatom,199775262375uatom
Binary Holdings,0.000983,445000000uatom,197454226889uatom
Huobi,0.000970,1000000uatom,194791822745uatom
ChainUp,0.000937,1000000uatom,188236562040uatom
Twinstake-Validator,0.000925,1000000uatom,185755532926uatom
Just-Mining,0.000909,1998800uatom,182509515589uatom
Smart Stake,0.000903,1556000000uatom,181369452594uatom
Stakely.io,0.000896,63000000uatom,179861375873uatom
Citizen Cosmos,0.000894,49267145uatom,179532906660uatom
in3s.com,0.000860,1000000uatom,172756052139uatom
PUPM√òS,0.000858,10000000uatom,172318198745uatom
Oni ‚õ©Ô∏è,0.000824,107242471uatom,165553309720uatom
coinhall.org üìàüîÑ,0.000813,1000000uatom,163240670312uatom
Klub Staking,0.000809,1000000uatom,162511558181uatom
Ztake.org,0.000808,149985000uatom,162216138942uatom
cros-nest,0.000783,1147900001uatom,157300079455uatom
jabbey,0.000777,0uatom,156022376925uatom
Easy 2 Stake,0.000775,9678026666uatom,155644560347uatom
GATA DAO,0.000766,77000000uatom,153869293284uatom
commercio.network,0.000760,3948000000uatom,152632502010uatom
NEOPLY,0.000747,1000000uatom,150112149979uatom
Blocks United | blocksunited.com,0.000747,499950008uatom,149945377560uatom
KIRA Staking,0.000728,1000000uatom,146264775213uatom
Silk Nodes,0.000702,1000000uatom,141042890744uatom
CryptoCrew Validators ‚úÖ,0.000685,1030750000uatom,137644554458uatom
AIR DROP STATION,0.000685,1000000uatom,137478506297uatom
Moonstake,0.000684,999600uatom,137455495211uatom
Stargaze,0.000681,25133644555uatom,136741951164uatom
CEX.IO ‚ù§Ô∏èüá∫üá¶,0.000668,1000000uatom,134118506615uatom
danku_zone w/ DAIC,0.000665,1100000uatom,133643294940uatom
BitValidator,0.000665,1000000uatom,133592971717uatom
blockscape,0.000650,6887546955uatom,130498219625uatom
Ping,0.000645,15600000000uatom,129580342932uatom
Castlenode,0.000624,1000000uatom,125364241836uatom
debo-validator,0.000611,24329999929uatom,122803501636uatom
Chainflow,0.000602,35617872936uatom,120821845430uatom
Any Labs,0.000601,999700uatom,120661486947uatom
Terra Nodes,0.000598,1880733950uatom,120063967212uatom
MANTRA DAO,0.000565,887822410uatom,113419124558uatom
Chill Validation,0.000550,380654867uatom,110515524315uatom
Fanfury | fury.fan | Stake to Win,0.000543,1000000uatom,109059541432uatom
Coinbase Cloud,0.000539,18960092871uatom,108287426453uatom
IcyCRO üßä,0.000529,2888465567uatom,106213887337uatom
stake.systems,0.000529,5112000000uatom,106184759848uatom
zoomerlabs,0.000506,502000000uatom,101594685151uatom
TienThuatToan Capital,0.000502,1000000uatom,100814646791uatom
Compass,0.000481,15267561312uatom,96573102823uatom
a41,0.000470,70983000000uatom,94424951424uatom
StakeSeeker by BTCS,0.000465,100000uatom,93339107961uatom
CROSSTECH,0.